### PR TITLE
Refine hex.info task output

### LINKED ISSUE #171: Improve hex.info task output
Currently `mix hex.info httpoison` gives the following output:
```
httpoison
  Config: {:httpoison, "~> 0.8.0"}
  Releases: 0.8.0, 0.7.5, 0.7.4, 0.7.3, 0.7.2, 0.7.1, 0.7.0, 0.6.2, 0.6.1, 0.6.0, 0.5.0, 0.4.3, 0.4.2, 0.4.1, 0.4.0, 0.3.2, 0.3.1, 0.3.0, 0.2.0

  Maintainers: Eduardo Gurgel Pinho
  Licenses: MIT
  Links:
    Github: https://github.com/edgurgel/httpoison

Yet Another HTTP client for Elixir powered by hackney
```
There are few things I think we could improve:
  1. Do not show first line: `httpoison` (seems very redundant, this what user provides as argument)
  2. Show only last 10 (other number?) releases
  3. Put description to the top (it is always very concise)

@ericmj, @josevalim, what are your thoughts?

### PR DESCRIPTION
**Pull request by lexmag**
Closes #171.
```elixir
# mix hex.info httpoison
Yet Another HTTP client for Elixir powered by hackney

Config: {:httpoison, "~> 0.8.0"}
Releases: 0.8.0, 0.7.5, 0.7.4, 0.7.3, 0.7.2, 0.7.1, 0.7.0, 0.6.2, 0.6.1, 0.6.0, ...

Maintainers: Eduardo Gurgel Pinho
Licenses: MIT
Links:
  Github: https://github.com/edgurgel/httpoison
```
```elixir
# mix hex.info httpoison 0.7.2
Config: {:httpoison, "~> 0.7.2"}
Documentation at: http://hexdocs.pm/httpoison/0.7.2/
Dependencies:
  hackney: ~> 1.3.1
```

### CODE REVIEW

File: lib/mix/tasks/hex/info.ex [hunk 1]
```diff
@@ -34,10 +34,8 @@ defmodule Mix.Tasks.Hex.Info do
   defp general() do
     Hex.Shell.info "Hex:    v#{Hex.version}"
     Hex.Shell.info "Elixir: v#{System.version}"
-    Hex.Shell.info "ERTS:   v#{:erlang.system_info(:version)}"
-    Hex.Shell.info ""
-    Hex.Shell.info "Built with: Elixir v#{Hex.elixir_version} and ERTS v#{Hex.erts_version}"
-    Hex.Shell.info ""
+    Hex.Shell.info "ERTS:   v#{:erlang.system_info(:version)}\n"
+    Hex.Shell.info "Built with: Elixir v#{Hex.elixir_version} and ERTS v#{Hex.erts_version}\n"
```
Comment (ericmj) [inline lib/mix/tasks/hex/info.ex]: If we keep them all together maybe we should align all of them.

Comment (lexmag) [inline lib/mix/tasks/hex/info.ex]: There is no change to previous formatting, I've just added `\n` at the end instead of extra `Hex.Shell.info ""`.

Comment (ericmj) [inline lib/mix/tasks/hex/info.ex]: Okay, i missed the newline.

**Comment by ericmj** [comment]
:heart:

**Pull request merged**